starknet-foundry/crates/forge/tests/e2e/collection.rs

```diff
@@ -18,39 +18,27 @@ fn complex_structure() {
         .current_dir(&temp)
         .assert()
         .success()
-        .stdout_matches(indoc! {r#"Collected 21 test(s) and 11 test file(s)
-        Running 1 test(s) from fob.cairo
-        [PASS] fob::fob::test_fob
-        Running 2 test(s) from src/fab/fab_impl.cairo
-        [PASS] fab_impl::fab_impl::test_fab
-        [PASS] fab_impl::fab_impl::test_how_does_this_work
-        Running 3 test(s) from src/fab/fibfabfob.cairo
-        [PASS] fibfabfob::fibfabfob::test_fib
-        [PASS] fibfabfob::fibfabfob::test_fob
-        [PASS] fibfabfob::fibfabfob::test_fab
-        Running 1 test(s) from src/fab.cairo
-        [PASS] fab::fab::test_simple
-        Running 3 test(s) from src/fib.cairo
-        [PASS] fib::fib::test_fib
-        [PASS] fib::fib::test_fob_in_fib
-        [PASS] fib::fib::test_fab_in_fib
-        Running 3 test(s) from src/fob/fibfabfob.cairo
-        [PASS] fibfabfob::fibfabfob::test_fib
-        [PASS] fibfabfob::fibfabfob::test_fob
-        [PASS] fibfabfob::fibfabfob::test_fab
-        Running 1 test(s) from src/fob/fob_impl.cairo
-        [PASS] fob_impl::fob_impl::test_fob
-        Running 1 test(s) from src/fob.cairo
-        [PASS] fob::fob::test_simple
-        Running 3 test(s) from src/lib.cairo
+        .stdout_matches(indoc! {r#"Collected 16 test(s) and 4 test file(s)
+        Running 12 test(s) from src/lib.cairo
```

Comment:
Is it possible to print the package name in this case? This message suggests that all the tests are in the src/lib.cairo